### PR TITLE
fix async errors

### PR DESCRIPTION
**Pull request by gorakong**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -26,19 +28,16 @@ class App extends React.Component {
       type: 'GET',
```
Comment (JayBo7) [inline client/src/components/App.jsx:26]: is this supposed to be ajax?

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -60,17 +59,21 @@ class App extends React.Component {
...
 
-    return rating;
+    this.setState({
+      reviews,
+      rating,
+    });
   }
 
   render() {
     const { reviews } = this.state;
```
Comment (JayBo7) [inline client/src/components/App.jsx]: you can just write this as const { reviews, ratings } instead of two lines

**Review by JayBo7 — CHANGES_REQUESTED** [review]
Think you're missing the $.ajax() function

File: client/src/components/App.jsx (hunk 1 above)
Comment (gorakong) [reply client/src/components/App.jsx:26]: yup! i think it's just hidden in the preview??

File: client/src/components/App.jsx (hunk 2 above)
Comment (gorakong) [reply client/src/components/App.jsx]: oh good suggestion. I have made the change!

**Comment by gorakong** [comment]
@JayBo7  thanks! i have made the appropriate changes.

File: client/src/components/App.jsx [hunk 3]
```diff
@@ -60,8 +59,7 @@ class App extends React.Component {
     });
 
     this.setState({
-      reviews,
-      rating,
+      reviews, rating,
     });
```
Comment (JayBo7) [inline client/src/components/App.jsx:62]: this isn't the right area

Comment (gorakong) [reply client/src/components/App.jsx:62]: the red parts indicate deleted lines..

Comment (JayBo7) [reply client/src/components/App.jsx:62]: Line 67-68 of app.jsx. Sorry about the confusion

Comment (gorakong) [reply client/src/components/App.jsx:62]: ohh i see. i'll fix it now

**Review by JayBo7 — APPROVED** [review]
Cool

**Pull request merged**